docs/admin/hardware.rst

```diff
@@ -0,0 +1,65 @@
+
+Recommended hardware
+====================
+
+.. include:: ../includes/top-warning.rst
+
+Qubes OS hardware requirements
+------------------------------
+
+In order to install and use SecureDrop Workstation, you will need a Qubes-Compatible computer with the following speifications:
+
+- 64-bit Intel or AMD processor with virtualization support
+- a minimum of 16GB RAM (32GB recommended for production use)
+- sufficient disk space for the Qubes OS base install and SecureDrop Workstation VMs (a 128GB or greater SSD is recommended)
+
+More information on hardware compatibility can be found on the `Qubes OS System Requirements <https://www.qubes-os.org/doc/system-requirements/>`_ page, and information on specific systems can be found via the `hardware compatibility list <https://www.qubes-os.org/hcl/>`_.
+
+In order to print submissions, a supported non-networked printer is required. Supported models currently include:
+
+- TBD
+- Also TBD
+
+More printer options will be added in future releases.
+
+Lenovo ThinkPad T480
+--------------------
+The ThinkPad T480 is a viable option for SecureDrop Workstation, and also has the advantage of being used heavily by the SecureDrop development team. If you plan to use it, you should follow the instructions below to ensure that the BIOS is up to date before proceeding with the installation:
```

Comment:
I wouldn't soft-pedal this, we can just say something like "We currently recommend using the Thinkpad T480 for the SecureDrop Workstation, and are using it during testing, development, and quality assurance."

Reply:
No other references to "we" in install or hardware docs, only "you". Will reword.